CoreFunctionality/src/au/gov/asd/tac/constellation/functionality/whatsnew/whatsnew.txt

```diff
@@ -1,6 +1,9 @@
 == 3030-12-31 Getting Started
 <p>If you're new to CONSTELLATION, read the <a href="" helpId="au.gov.asd.tac.constellation.functionality.introduction">introduction</a>.</p>
 
+== 2020-08-28 Split Nodes Plugin
+<p>A new option is added to split nodes with duplicate transactions.</p>
```

Comment:
The first interpretation that came to mind when reading this was that it splits nodes that have duplicate transactions rather than duplicating the transactions of the split node for all the newly created nodes. Might be worth slightly altering the entry just to make the latter (correct) interpretation of the functionality clearer.

Reply:
Fixed